src/main/resources/db/liquibase/changelog/1-database-init.xml

```diff
@@ -0,0 +1,128 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                   http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <preConditions onFail="HALT">
+        <not>
+            <tableExists tableName="databasechangelog" schemaName="public"/>
+        </not>
+    </preConditions>
+
+    <changeSet author="arfedorov" id="1-database-init`">
+        <createTable tableName="article">
+            <column autoIncrement="true" name="id" type="INTEGER">
+                <constraints primaryKey="true" primaryKeyName="article_pkey"/>
```

Comment:
Please think which columns could be null, and which are optional. Add nullable="false" to the mandatory fields

Reply:
Same for all other tables